assets/src/apps/bibliography/Bibliography.tsx

```diff
@@ -326,39 +324,39 @@ const Bibliography: React.FC<BibliographyProps> = (props: BibliographyProps) =>
   );
 
   return (
-    <div className="resource-editor row">
-      <div className="col-span-12">
-        <Banner
-          dismissMessage={(msg) => setMessages(messages.filter((m) => msg.guid !== m.guid))}
-          executeAction={(message, action) => action.execute(message)}
-          messages={messages}
-        />
-        <div className="d-flex justify-content-start">
-          <button type="button" className="btn btn-link" onClick={() => onPlainCreateOrEdit()}>
-            <i className="fas fa-solid la-plus"></i> Add Entry
-          </button>
-          {createEntryDropdown}
-        </div>
-        <hr className="mb-4" />
+    <React.StrictMode>
+      <ErrorBoundary>
+        <ModalDisplay />
+
+        <div className="resource-editor row">
+          <div className="col-span-12">
+            <Banner
+              dismissMessage={(msg) => setMessages(messages.filter((m) => msg.guid !== m.guid))}
+              executeAction={(message, action) => action.execute(message)}
+              messages={messages}
+            />
+            <div className="d-flex justify-content-start">
+              <button type="button" className="btn btn-link" onClick={() => onPlainCreateOrEdit()}>
+                <i className="fas fa-solid la-plus"></i> Add Entry
+              </button>
+              {createEntryDropdown}
+            </div>
+            <hr className="mb-4" />
 
-        {pagingOrPlaceholder}
+            {pagingOrPlaceholder}
 
-        {bibEntryViews}
+            {bibEntryViews}
 
-        {totalCount > 0 ? pagingOrPlaceholder : null}
-      </div>
-    </div>
+            {totalCount > 0 ? pagingOrPlaceholder : null}
+          </div>
+        </div>
+      </ErrorBoundary>
+    </React.StrictMode>
   );
 };
 
 function defaultPaging() {
   return { offset: 0, limit: PAGE_SIZE };
 }
 
-const BibliographyApp: React.FC<BibliographyProps> = (props) => (
```

Comment:
Some apps were being unnecessarily wrapped twice in redux provider. The call to registerApplication wraps these components with a specified redux store so this is unnecessary